app/assets/stylesheets/therapists/_show.scss

```diff
@@ -0,0 +1,16 @@
+.show-container {
```

Comment:
will you make your classes specific to your page, aka 'therapist-show-container'